README.md

```diff
@@ -62,14 +62,27 @@ By default the running app can be found at `localhost:3000`.
     ```
 
     OR prepare .env with default content:
-    
+
     ```
     CLIENT_ID
     CLIENT_SECRET
     SESSION_SECRET
     TARGET_APP
+    API_URL
+    CUSTOM_MUNICIPALITY_OPTIONS
     ```
 
+    Environment's variable guideline:
+
+    - `API_URL`:
+      Custom config to replace global application's api URL. Expected value is valid URL string.
+
+    - `CUSTOM_MUNICIPALITY_OPTIONS`:
+      Config for custom municipalities. Expected value is array of municipalities: `['example_city']`.
```

Comment:
I don't think anyone wants to add cities with `_` :)

Reply:
might just try Tampere :D